Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -38,12 +38,18 @@ func testEnvironmentIsSigned() -> Bool {
 
 func testEnvironmentSupportsWhenPasscodeSet() -> Bool {
     if let simulatorVersionInfo = ProcessInfo.processInfo.environment["SIMULATOR_VERSION_INFO"],
-        simulatorVersionInfo.contains("iOS 13") || simulatorVersionInfo.contains("tvOS 13")
+        simulatorVersionInfo.contains("iOS 13")
+        || simulatorVersionInfo.contains("tvOS 13")
+        || simulatorVersionInfo.contains("iOS 14")
+        || simulatorVersionInfo.contains("tvOS 14")
```

Comment:
Before this PR gets out of Draft I need to double-check iOS 14 and tvOS 14.

Reply:
Looks like iOS 14 and tvOS 14 work! c7edf5d

Reply:
Weiiirrrdddddd

Reply:
Yeah that was seriously unexpected.